src/modules.js

```diff
@@ -113,6 +113,7 @@ var LibraryManager = {
         'library_fs.js',
         'library_memfs.js',
         'library_tty.js',
+        'library_pipefs.js',
```

Comment:
Unfortunately, moving `library_pipefs.js` above the `library_fs.js` here seems not to change the order in which postsets are emitted in the resulting js. Reversing `__deps` to `$FS -> $PIPEFS` seems not to change it too. So it works, but no one know how. :(